tbx/core/blocks.py

```diff
@@ -232,6 +132,19 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/video_block.html"
 
 
+class ExternalStoryEmbedBlock(WebstoryExternalStoryEmbedBlock):
+    """This code is no longer in use, unfortunately an early migration depends explicitly on it & migrations cannot be simply squashed to get around this"""
+
+    # Work around due to Attribute Error https://github.com/torchbox/wagtail-webstories/pull/12
+    def get_prep_value(self, value):
+        if value is None:
+            return ""
+        elif isinstance(value, str):
+            return value
+        else:
+            return value.url
```

Comment:
This can be removed completely, since

- the PR in question was actually merged
- we're not using the block

**Suggestion**

Replace it with 

```python
    pass
```